src/mount.fuse.ceph

```diff
@@ -39,8 +39,11 @@ def ceph_options_compat(device):
     return [ 'ceph.' + opt for opt in device.split(',') ]
 
 def fs_options(opts, ceph_opts):
-    # strip out noauto and _netdev options; libfuse doesn't like it
-    strip_opts = ['defaults', 'noauto', '_netdev']
+    # - strip out noauto and _netdev options; libfuse doesn't like it
+    # - nofail option is also not recognized by libfuse.
+    #   Starting with fuse 3.2.2 the option is also ignored by mount.fuse, see
+    #   https://github.com/libfuse/libfuse/commit/a83cd72f641671b71b8268b1765e449cae071f3e
+    strip_opts = ['defaults', 'noauto', '_netdev', 'nofail']
```

Comment:
I think this approach is correct. Just one question - In the redmine tracker you put

```
 I'll submit a patch that does the same on our side, for versions of fuse < 3.2.2
```

So, I expected to see some form for version check somewhere. What am I missing?

Reply:
I should add comment or edit that message on the tracker. The thing is, when implementing this, I realized that I should not be checking the version as the fix from the library is only affecting users of their supplied `mount.fuse` helper. In our case, we won't be calling through their mount helper, we are using our own. As a result, the correct approach should be filtering this option unconditionally.

Reply:
Makes sense. Thx for explaining.